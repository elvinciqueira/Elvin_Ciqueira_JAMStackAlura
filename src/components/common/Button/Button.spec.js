/* eslint-disable no-undef */
import React from 'react'
import {Button} from './'

test('it renders', () => {
  const tree = rendererCreateWithTheme(<Button />).toJSON()
  expect(tree).toMatchSnapshot()
  expect(tree).toHaveStyleRule('font-weight', 'bold')
})
