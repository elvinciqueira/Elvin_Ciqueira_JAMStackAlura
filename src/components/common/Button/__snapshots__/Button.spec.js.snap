// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it renders 1`] = `
.c0 {
  font-weight: bold;
  font-size: 1.75rem;
  background: none;
  border: 1px solid #E9C46A;
  border-radius: 50%;
  height: 46px;
  width: 46px;
}

<button
  className="c0"
/>
`;
